homeassistant/components/cover/command_line.py

```diff
@@ -154,3 +159,11 @@ def close_cover(self, **kwargs):
     def stop_cover(self, **kwargs):
         """Stop the cover."""
         self._move_cover(self._command_stop)
+
+    def set_cover_position(self, **kwargs):
+        """Set the cover position."""
+        if self.available:
```

Comment:
It doesn't seem like this entity is overriding the available property. The default value is `True` so there is no need for this.